.kokoro/stage.sh

```diff
@@ -29,7 +29,7 @@ setup_environment_secrets
 create_settings_xml_file $MAVEN_SETTINGS_FILE
 
 # run unit tests
-./mvnw verify -B -V -P disable-integration-tests
+./mvnw install -B -V
```

Comment:
`install` will run integration tests. Is that your intent here?

Reply:
Fixed, same as above.